config

```diff
@@ -25,3 +19,5 @@ export RH_CHE_DOCKER_IMAGE_VERSION=${RH_CHE_DOCKER_IMAGE_VERSION:-${DEFAULT_RH_C
 export DOCKER_HUB_USER=${DOCKER_HUB_USER:-${RHCHEBOT_DOCKER_HUB_USER}}
 export DOCKER_HUB_PASSWORD=${DOCKER_HUB_PASSWORD:-${RHCHEBOT_DOCKER_HUB_PASSWORD}}
 export DOCKER_HUB_NAMESPACE=${DOCKER_HUB_NAMESPACE:-${RH_CHE_DOCKER_HUB_NAMESPACE}}
+
+export CHE_DOCKER_BASE_IMAGE=eclipse/che-server:5.17.0-centos
```

Comment:
the problem is that current `5.18.0-snapshot` has the docker tag `nightly-centos` but after the next release maven version will change to `5.19.0-snapshot` but docker tag will remain unchanged `nightly-centos`. So at the moment we can't really sync this. But tagged version are using the version so it would work if we had a 5.18.0-SNAPSHOT-centos upstream but we don't have at the moment. I think it is simple enough with that so we don't over-engineering

Reply:
OK